From 7e7392c209f2c70dec20b791d611e61e6bf71144 Mon Sep 17 00:00:00 2001
From: Katsuya Matsubara <matsu@igel.co.jp>
Date: Tue, 19 Jun 2012 19:25:15 +0900
Subject: [PATCH 46/80] rcarvin: add the YUYV format as an extra supported color

VIN could convert any (accepted) input colors into YUYV.
This adds the YUYV format into the extra colors list.

Signed-off-by: Katsuya Matsubara <matsu@igel.co.jp>
---
 drivers/media/video/rcarvin.c |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/rcarvin.c b/drivers/media/video/rcarvin.c
index 15dfaf2..137dda5 100644
--- a/drivers/media/video/rcarvin.c
+++ b/drivers/media/video/rcarvin.c
@@ -1063,6 +1063,13 @@ static const struct soc_mbus_pixelfmt rcar_vin_formats[] = {
 		.order			= SOC_MBUS_ORDER_LE,
 	},
 	{
+		.fourcc			= V4L2_PIX_FMT_YUYV,
+		.name			= "YUYV",
+		.bits_per_sample	= 16,
+		.packing		= SOC_MBUS_PACKING_NONE,
+		.order			= SOC_MBUS_ORDER_LE,
+	},
+	{
 		.fourcc			= V4L2_PIX_FMT_RGB565,
 		.name			= "RGB565",
 		.bits_per_sample	= 16,
-- 
1.7.0.4

